### PR TITLE
Adds base content

### PR DESCRIPTION
**Pull request by JoshVanL**
Signed-off-by: JoshVanL <vleeuwenjoshua@gmail.com>

### CODE REVIEW

File: content/docs/contributing/external-issuers.md [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+
+Once a signed certificate has been gathered by the issuer controller, it then
+updates the status of the `CertifiateRequest` resource with the signed
+certificate. It is then important to then update the condition status of that
+resource to a ready state, as this is what is used to signal to higher order
+controllers, such as the `Certificate` controller, that the resource is ready to
+be consumed. Conversely, if the `CertificateRequest` fails, it is as important
+to mark the resource as such, as this will also be used to signal to higher
+order controllers. You can read the valid condition sates
+[here](../../concepts/certificaterequest/index.html).TODO: add '#Conditions'
```
Comment (munnerz) [inline content/docs/contributing/external-issuers.md]: I think usually you just omit the `index.html` altogether here when linking - can we do the same here, and link to `../../concepts/certificaterequests/`? I think that passes checks still... 😄

File: content/docs/tutorials/_index.md [hunk 2]
```diff
@@ -0,0 +1,29 @@
+---
+title: "Tutorials"
+linkTitle: "tutorials"
```
Comment (munnerz) [inline content/docs/tutorials/_index.md]: The linkTitle is used in the sidebar, which means that currently all of our sidebar navigation is lowercase.. can you go through and update these?

Comment (JoshVanL) [reply content/docs/tutorials/_index.md]: :cry:

File: content/en/docs/contributing/external-issuers.md [hunk 3]
```diff
@@ -8,7 +8,7 @@ type: "docs"
 cert-manager offers a number of [core issuer
 types](../../configuration/) that represent certificate authorities
 that can sign certificates when requested. As of v0.11, cert-manager also
-supports out-of-tree external issuers natively, and treats them the same as
+supports out-of-tree external issuers naively, and treats them the same as
```
Comment (munnerz) [inline content/en/docs/contributing/external-issuers.md]: Naively isn't correct 😅

File: content/en/docs/installation/kubernetes.md [hunk 4]
```diff
@@ -6,7 +6,7 @@ type: "docs"
 ---
 
 cert-manager runs within your Kubernetes cluster as a series of deployment
-resources. It utilises
+resources. It utilities
```
Comment (munnerz) [inline content/en/docs/installation/kubernetes.md]: I think this should have been 'utilizes'?

File: content/en/docs/installation/kubernetes.md [hunk 5]
```diff
@@ -92,9 +92,9 @@ kubectl create clusterrolebinding cluster-admin-binding \
 As an alternative to the YAML manifests referenced above, we also provide an
 official Helm chart for installing cert-manager.
 
-### Pre-requisites
+### Prerequisites
 
-- Helm_ and Tiller installed (or alternatively, use `Tillerless Helm v2`_)
+- Helm and Tiller installed (or alternatively, use `Tillerless Helm v2`z)
```
Comment (munnerz) [inline content/en/docs/installation/kubernetes.md]: Underscores were there to make these words links - also the 'z' at the end seems like a typo

File: content/en/docs/installation/openshift.md [hunk 6]
```diff
@@ -7,7 +7,7 @@ type: "docs"
 
 cert-manager supports running on OpenShift in a similar manner to [Running on
 Kubernetes](../kubernetes/).  It runs within your OpenShift cluster as a series
-of deployment resources. It utilises
+of deployment resources. It utilities
```
Comment (munnerz) [inline content/en/docs/installation/openshift.md]: Utilizes

File: content/en/docs/release-notes/release-notes-0.2.md [hunk 7]
```diff
@@ -7,7 +7,7 @@ type: "docs"
 
 ## Changelog since v0.1.0
 ### Action Required
-- Move to 'jetstack' organisation. Action required: this will require updating your existing deployments to point to the new image repository, as new tags will not be pushed to the old 'jetstackexperimental/cert-manager-controllerrepository. Ahelm upgrade` should take care of this. (#145, @munnerz)
+- Move to 'jetstack' organization. Action required: this will require updating your existing deployments to point to the new image repository, as new tags will not be pushed to the old 'jetstackexperimental/cert-manager-controllerrepository. A helm upgrade` should take care of this. (#145, @munnerz)
```
Comment (munnerz) [inline content/en/docs/release-notes/release-notes-0.2.md]: Not you, but there is a backtick here for no reason after the word 'upgrade' causing this to render weirdly

File: .spelling [hunk 8]
```diff
@@ -1,5 +1,498 @@
...
+acme-dns
+acme-dns01-provider
+acmedns.go
+acme-http01-edit-in-place
+acme-http01-ingress-class
+acmeorders
+acmesolver
+ACMEv1
+ACMEv2
+acoshift
```
Comment (munnerz) [inline .spelling]: What is acoshift? 🤔

File: .spelling [hunk 9]
```diff
@@ -1,5 +1,498 @@
-Kubernetes
+100m
```
Comment (munnerz) [inline .spelling]: It seems like quite a few here are from code snippets? Our spellchecker should skip text in backticks, which I think should make a lot of these not necessary?

File: .spelling [hunk 10]
```diff
@@ -1,5 +1,498 @@
...
+glog
+gordonbondon
+Gottwald
+grep
+groner
+GuillaumeSmaha
+gurvindersingh
+hardcoded
+Hashicorp
 HashiCorp
```
Comment (munnerz) [inline .spelling]: We should use one or the other, not both 🙂

File: .spelling [hunk 11]
```diff
@@ -1,5 +1,498 @@
...
+INWX
+io
+ip-sans
+ipv6
+isCA
+issuerRef.group
+issuerRef.kind
+issuer.spec.acme.solvers
+istio
+Istio
```
Comment (munnerz) [inline .spelling]: Capital or not?

**Comment by munnerz** [comment]
Going to follow up with additional PRs on this - merging so we can both work on it 😄 

/lgtm

**Pull request merged**